test/handlers.jl

```diff
@@ -106,7 +106,7 @@
 
         @testset "Sample Usage w/ File" begin
             filename = tempname()
-            info("Path to log file: $filename")
+            Compat.@info("Path to log file: $filename")
```

Comment:
Personal preference but I prefer doing `using Compat: @info` so I don't have to have `Compat.*` peppered throughout the code. Will make dropping 0.6 support later easier.